docs/leakscope.md

```diff
@@ -1,6 +1,8 @@
 @property {Boolean} can-component.prototype.leakScope leakScope
 @parent can-component.deprecated
 
+@deprecated Pass your values instead.
```

Comment:
Same; examples are helpful to people upgrading / cleaning up tech debt. 😊

Reply:
I already fixed that

Reply:
Done!